tagmanager/src/tm_source_file.h

```diff
@@ -14,14 +14,12 @@
 #include <glib.h>
 #include <glib-object.h>
 
+#include "tm_parser.h"
```

Comment:
I tested it over existing installation so I missed this. The tm_parser.h file can easily be made part of the installation so if others find the API "break" acceptable I'll update the patch to install the file.